pgspecial/iocommands.py

```diff
@@ -10,7 +10,7 @@
 from os.path import expanduser
 from .namedqueries import NamedQueries
 from . import export
-from .main import show_extra_help_command, special_command
+from .main import show_extra_help_command, special_command, PARSED_QUERY
```

Comment:
`PARSED_QUERY` doesn't seem to be used, you can probably remove this import. `black` should have done that for you.